src/util/request-animation-frame.js

```diff
@@ -11,5 +11,5 @@ export default (
     window.mozRequestAnimationFrame ||
     window.oRequestAnimationFrame ||
     window.msRequestAnimationFrame ||
-    ((callback, element) => setTimeout(callback, 1000 / 60))
```

Comment:
no need to change the existing implementation.

Reply:
Don't be anal man 😂 

It was an unused arg.

Reply:
Love it or hate it, I don't care.

Reply:
Mostly bemusement at the insistence for keeping an unused arg around.

Reply:
This isn't changing implementation, it's removing an unused arg. 🤷‍♂ 

Nothing else changed.

Reply:
There's a lot of quirks in old code, who knows, maybe it's used in some weird way somewhere, by someone. If we keep it, that will definitely **not** cause issues. No need to introduce a change to this function.

Reply:
In a language like JS that's literally impossible. =)